### PR TITLE
Restore extra info

### PR DESCRIPTION
**Pull request by bramstoeller**

### CODE REVIEW

File: src/power_grid_model_io/config/excel/vision_en.yaml [hunk 1]
```diff
@@ -137,14 +146,15 @@ grid:
       r_grounding_to: Re2
       x_grounding_to: Xe2
       extra:
-        - Name
+        - Number
```
Comment (petersalemink95) [inline src/power_grid_model_io/config/excel/vision_en.yaml:149]: transformer used to have Subnumber

Comment (petersalemink95) [reply src/power_grid_model_io/config/excel/vision_en.yaml:149]: (and Node.Number)

File: src/power_grid_model_io/config/excel/vision_en.yaml [hunk 2]
```diff
@@ -244,8 +255,10 @@ grid:
       r_grounding_to: 0
       x_grounding_to: 0
       extra:
-        - Name
+        - Node.Number
```
Comment (petersalemink95) [inline src/power_grid_model_io/config/excel/vision_en.yaml:258]: I guess the same here (Node.Number / Subnumber)

File: src/power_grid_model_io/config/excel/vision_en.yaml (hunk 1 above)
Comment (bramstoeller) [reply src/power_grid_model_io/config/excel/vision_en.yaml:149]: I think that's only for transformers created from the Transformer loads table.

File: src/power_grid_model_io/config/excel/vision_en.yaml (hunk 2 above)
Comment (bramstoeller) [reply src/power_grid_model_io/config/excel/vision_en.yaml:258]: Whet do you mean? Here it does say Node.Number and Subnumber, right?

**Review by petersalemink95 — APPROVED** [review]

**Pull request merged**